Change entries which will most likely be changed by later
patches into remarks or empty assignments.

Index: etc/make.conf
===================================================================
--- etc.orig/make.conf
+++ etc/make.conf
@@ -22,11 +22,14 @@
 #
 # Example:
 #USE="X gtk gnome -alsa"
+# USE flags sorted alphabetically for easier maintenance.
+USE="
+"
 
 # Host Setting
 # ============
 #
-CHOST="i686-pc-linux-gnu"
+#CHOST="i686-pc-linux-gnu"
 
 # Host and optimization settings 
 # ==============================
@@ -66,10 +69,12 @@ CHOST="i686-pc-linux-gnu"
 #
 #CFLAGS="-mcpu=athlon-xp -O2 -pipe"
 #CFLAGS="-march=pentium3 -O2 -pipe"
+# Avoid -O0 and -s in CFLAGS.
 
 # If you set a CFLAGS above, then this line will set your default C++ flags to
 # the same settings.
 #CXXFLAGS="${CFLAGS}"
+CXXFLAGS="${CFLAGS}"
 
 # Advanced Masking
 # ================
@@ -145,6 +150,8 @@ CHOST="i686-pc-linux-gnu"
 #     concern that they will be deleted by rsync updates. Default is not
 #     defined.
 #PORTDIR_OVERLAY=/usr/local/portage
+# Add more overlays (from "layman").
+#source /usr/local/portage/layman/make.conf
 
 # Fetching files 
 # ==============
@@ -277,6 +284,9 @@ CHOST="i686-pc-linux-gnu"
 # values and their respective meanings.
 #FEATURES="ccache distcc installsources \
 # splitdebug test userpriv usersandbox"
+# FEATURES sorted alphabetically for easier maintenance.
+FEATURES="
+"
 
 # CCACHE_SIZE and CCACHE_DIR are used to control the behavior of ccache, and
 #     and are only used if "ccache" is in FEATURES.
@@ -303,7 +313,7 @@ CHOST="i686-pc-linux-gnu"
 # PORTAGE_ELOG_CLASSES: selects messages to be logged, possible values are:
 #                          info, warn, error, log, qa, *
 #                       Warning: commenting this will disable elog
-PORTAGE_ELOG_CLASSES="log warn error"
+#PORTAGE_ELOG_CLASSES="warn error log"
 
 # PORTAGE_ELOG_SYSTEM: selects the module(s) to process the log messages. Modules
 #                      included in portage are (empty means logging is disabled):
